OpenQA/Isotovideo/Utils.pm

```diff
@@ -131,6 +132,57 @@ sub checkout_git_refspec {
     calculate_git_hash($dir);
 }
 
+=head2 handle_generated_assets
+
+Handles the assets generated by the test depending on status and test
+configuration variables.
+
+=cut
+
+sub handle_generated_assets {
+    my ($command_handler, $return_code, $clean_shutdown) = @_;
```

Comment:
The `$return_code` should be passed as scalar ref (and the assignments should change the value referred by `$return_code`) so the remaining code in `isotovideo` has the updated value.

Reply:
So you would call it like `handle_generated_assets($command_handler, \$return_code, $clean_shutdown);` and do the assignments like `$$return_code = 1`.

Reply:
changed to not take the variable at all but return a value which is used within the calling scope together with a check if we should call the function at all.